### DISCUSSION TITLE
Building instructions with flexible tubes and submodels

### DISCUSSION THREAD

**Discussion by pepijndevos** [leozide/leocad#760] [Q&A]
I'm working on a model, but facing a few issues.

First of all, I'm using these thin smooth flexible tubes. I don't see them in the library, and am not sure how these flexible parts work?

Then when creating a the building instructions, the model clips out of the page, and more importantly, the submodel is inserted without any instructions.

I found here how to add the camera for the building instructions: https://github.com/leozide/leocad/issues/428#issuecomment-571757550

But how can I include building instructions for the subcomponent? I remember I've done it ages ago in another program, but no idea what or how. I used to know all these fancy tricks with callouts and arrows and what not but... it's been a looong time.

[scale.zip](https://github.com/leozide/leocad/files/7796901/scale.zip)

**Comment by leozide** [comment]
Flexible parts will have an icon in the top right corner of the preview, most of them are in the technic category. What's the part number of the tube you're trying to bend?

How are you saving the instructions? I was working on a layout editor but haven't been able to finish it yet.
